npm/ng-packs/packages/theme-shared/extensions/src/lib/components/extensible-table/extensible-table.component.html

```diff
@@ -24,23 +24,24 @@
 
   <ng-container *ngFor="let prop of propList; let i = index; trackBy: trackByFn">
     <ngx-datatable-column
+      *ngIf="prop.columnVisible(getInjected)"
```

Comment:
I would use Abp visible.  instead of if. It can work with observable and promise

https://github.com/abpframework/abp/blob/dev/npm/ng-packs/packages/theme-shared/src/lib/directives/visible.directive.ts

Reply:
Hi, changes has been applied.